testfiles-luatex/stop-link.tpf

```diff
@@ -300,24 +300,24 @@ BT
 ET
 endstream
 endobj
-21 0 obj
-<< /Type /Page /Contents 22 0 R /Resources 20 0 R /MediaBox [ 0 0 612 792 ] /Parent 16 0 R /Annots 25 0 R >>
+22 0 obj
+<< /Type /Page /Contents 23 0 R /Resources 21 0 R /MediaBox [ 0 0 612 792 ] /Parent 16 0 R /Annots 25 0 R >>
 endobj
 25 0 obj
-[ 18 0 R 24 0 R ]
+[ 18 0 R 20 0 R ]
 endobj
 18 0 obj
 << /Type /Annot /Subtype /Link /Border [0 0 1] /H /I /C [0.724708 0.4 0.717646] /Contents <FEFF00680074007400700073003A002F002F007400750067002E006F00720067> /A <</Type /Action /S /URI /URI (https://tug.org) >>  /Rect [ 179.547 691.108 230.555 705.056 ]
  >>
 endobj
-24 0 obj
-<< /Type /Annot /Subtype /Link /Border [0 0 1] /H /I /C [0.724708 0.4 0.717646] /Contents <FEFF00680074007400700073003A002F002F007700770077002E006C0061007400650078002D00700072006F006A006500630074002E006F00720067> /A <</Type /Action /S /URI /URI (https://www.latex-project.org) >>  /Rect [ 133.768 655.183 344.767 664.149 ]
+20 0 obj
+<< /Type /Annot /Subtype /Link /Border [0 0 1] /H /I /C [0.724708 0.4 0.717646] /Contents <FEFF00680074007400700073003A002F002F007700770077002E006C0061007400650078002D00700072006F006A006500630074002E006F00720067> /A <</Type /Action /S /URI /URI (https://www.latex-project.org) >>  /Rect [ 132.772 654.187 345.763 665.146 ]
```

Comment:
This seems to make the link bigger by 1pt in all directions. Do we know where this is coming from?

Reply:
Not sure. It is not related to the link code. My guess is the new output routine.  Btw: I think I will also the footnote-socket by default, that is a nice use case.